x/pool-incentives/keeper/keeper.go

```diff
@@ -58,29 +63,60 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", "x/"+types.ModuleName)
 }
 
+// CreatePoolGauges checks if the poolId is CLPoolType, if it is create one gauge,
+// otherwise create multiple gauges based on lockableDurations.
 func (k Keeper) CreatePoolGauges(ctx sdk.Context, poolId uint64) error {
```

Comment:
Please add a test for this method for both CL and non-CL pool

Reply:
I did not see this get added